src/storybookCommon/argTypes.ts

```diff
@@ -29,3 +29,23 @@ export const ophanComponentIdArgType = {
             '`rc_upsell_test1_variant`.',
     },
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Why is this required here? It looks like we should know the types of what we're returning?

Reply:
Good spot, it's not needed (it was left over from a previous iteration of this work).